files/config.define_roots.fig

```diff
@@ -18,17 +20,20 @@ thumbnails: {
 }
 
 [dev]
-locale: se_SV
+locale: "se_SV"
 
 # Overwrites of specific keys for the dev root
 [dev]
-locale: en_UK
+locale: "en_UK"
+
+[tests.%{dev,production}.anotherRoot.more.%{test,this}.aswell]
+thisWorks: ""
 
 # More specified overwrite
-[dev.thumbnails.small]
+[@.thumbnails.small]
 width: 40
 height: 30
 
 
-[dev prod]
-host: localhost
\ No newline at end of file
+[%{dev,prod}]
```

Comment:
Here we don't need to defined them again, as they were already defined in the top part of the file.